crates/cargo-util-schemas/src/schema.rs

```diff
@@ -0,0 +1,56 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::string::String;
+use toml::Value as TomlValue;
+#[derive(Serialize, Deserialize)]
+pub struct TomlValueWrapper(pub TomlValue);
+#[test]
+fn dump_schema() {
```

Comment:
Generally you want a newline between `use`, types, and functions.

I would not put an unrelated function between the type and its impl block

Reply:
Generally, I would expect a test to be at the bottom of a file.

In this case, I think `dump_schema` should likely live in
- `manifest/mod.rs` as its dumping that type
- an integration test for schemas in a function named for dealing with the manifest schema